website/static/js/logActionsList.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Created by cos-caner on 12/4/15.
+ */
+
+var logActions = {
```

Comment:
The only real downsides to this method are:
* This requires a centralized place that will have to be updated with addons, while most of the rest of the addon architecture is modular;
* If we get around to localizing strings, these will not be where many of the other strings are.

The second item is less important than the first, as we've not done amazingly well at ensuring even our Python strings will be ready for localization. However, if we can at least have a method of gathering log actions from individual addons rather than consolidating them here, that should make life a little easier for release management of addons.